### DISCUSSION TITLE
LDAP: Creation of LDAP record upon creation of user account

### DISCUSSION THREAD

**Discussion by dm-coding** [modoboa/modoboa#1227] [Ideas]
Hi all,

We'd like to use modoboa as a central dashboard where mail super-admins and domain admins can create users in the modoboa control panel and have user records created or pushed into the LDAP store. This simplifies the process of account creation for our end-users, since they wouldn't have to interact directly with our LDAP server. I understand that in many situations LDAP applications are designed to function as receivers of data from the LDAP auth store, but I've been part of several web-facing organizations which have used LDAP for single-sign in between web their services where this isn't the most appropriate model. Is there interest in this from the modoboa community? Would developers be interested in implementing this, or would you be interested in accepting a patch or a plugin? Great work on modoboa btw, we're really looking forward to migrating to it within our organization!

Thanks all!

**Comment by tonioo** [comment]
Such a feature would be really appreciated! A pull request would be even better ;-)

**Comment by blackyzero** [comment]
i am also looking for LDAP user database support.
Would it be able to be considered adding this feature in next release? :)

**Comment by tonioo** [comment]
@djcf Hi. Any news about this?
